### PR TITLE
Add download/push scripts for Ubuntu RTOS EKS-A image

### PR DESCRIPTION
**Pull request by abhay-krishna**
This PR adds the build scripts and logic for the Ubuntu RTOS image provided by Canonical.

Part of https://github.com/aws/eks-anywhere-internal/issues/2184

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release/staging-build.yml [hunk 1]
```diff
@@ -186,6 +186,13 @@ batch:
         variables:
           PROJECT_PATH: projects/brancz/kube-rbac-proxy
           CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/brancz.kube-rbac-proxy
+    - identifier: canonical_ubuntu
+      env:
+        type: ARM_CONTAINER
+        compute-type: BUILD_GENERAL1_SMALL
+        variables:
+          PROJECT_PATH: projects/canonical/ubuntu
+          CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/canonical.ubuntu
```
Comment (chrisdoherty4) [inline release/staging-build.yml:195]: What's in this repository?

**Review by chrisdoherty4 — APPROVED** [review]
/lgtm

File: release/staging-build.yml (hunk 1 above)
Comment (abhay-krishna) [reply release/staging-build.yml]: Removed it by setting REPO_NO_CLONE=true in Makefile.

**Pull request merged**